system-config/vagrant-reload.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2013 Aidan Nagorcka-Smith
+#
+# MIT License
```

Comment:
Did Aidan Nagorcka-Smith not add their copyright to this ruby script? Why are we adding it?